src/wazuh_testing/utils/configuration.py

```diff
@@ -344,6 +344,8 @@ def get_test_cases_data(data_file_path):
     test_cases_ids = []
 
     for test_case in test_cases_data:
+        if test_case['metadata'] is None:
```

Comment:
```suggestion
        if test_case.get('metadata') is None:
```

`test_case['metadata']` will raise an error if the key `metadata` is not present on the dict, I think it would be better to use the `.get()` function instead (it returns None if the key does not exist).

Reply:
Yes, would be better. Applied